### PR TITLE
[add] command to clone our tests into cypress folder

### PR DESCRIPTION
**Pull request by nh758**

### CODE REVIEW

File: lib/tasks/addTests.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+ */
+function checkFolder(done) {
+   console.log(`Check the tests folder exists`);
+   try {
+      if (fs.statSync(pathToIntegrationsFolder).isDirectory()) {
+         console.log("   ... done\n");
+         done();
+      }
+   } catch (e) {
+      console.log(`Could not find the path ${pathToIntegrationsFolder}`);
```
Comment (Hiro-Nakamura) [inline lib/tasks/addTests.js:106]: I think you should add `done(e)` here.

Or maybe a `process.exit(1)`  to exit the cli.

File: lib/tasks/addTests.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+ */
+function cloneRepos(done) {
+   const total = testsToAdd.length;
+   let testNumber = 1;
+   async.eachSeries(
+      testsToAdd,
+      (test, cb) => {
+         console.log(`\n git clone ${test} ${testNumber}/${total}`);
+         testNumber++;
+         const gitURL = `https://${token}github.com/digi-serve/${test}.git`;
```
Comment (Hiro-Nakamura) [inline lib/tasks/addTests.js:173]: is there any issue with adding `token` onto every url?  or should it be added only on the urls with the private repos?

**Review by Hiro-Nakamura — CHANGES_REQUESTED** [review]
this looks handy.

take a look at the exit issue.

File: lib/tasks/addTests.js (hunk 2 above)
Comment (nh758) [reply lib/tasks/addTests.js:173]: It's not needed for public repos, but it will still work. I tested it.

**Pull request merged**